NewSpring/ApollosProject
# Updated Bundle Id and App name

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR updates the bundle id and app name for the NewSpring iOS and Android versions of The Apollos Project. 

### How do I test this PR?
Check to see if both the android and iOS versions of the app run and have the name `newspringchurchapp`.
> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO:

- [ ] Closes [#ISSUE]

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
